### DISCUSSION TITLE
How can I uninstall video-subtitle-extractor?

### DISCUSSION THREAD

**Discussion by roland630** [YaoFANGUK/video-subtitle-extractor#182] [Q&A] [answered]
Hi there,
I just installed video-subtitle-extractor on my Windows system, but it crashed as soon as I ran it. I plan to uninstall and then reinstall it. Can you tell me how to uninstall it? Thank you!

**Comment by YaoFANGUK** [comment, marked as answer]
if you use .exe, just delete it.

If you use source code, please run ```conda env remove -n vseEnv```

**Reply by roland630** [reply]
Thanks for your response!
